packages/sites/genomics-site/webapp/wdkCustomization/js/client/components/homepage/VEuPathDBHomePage.tsx

```diff
@@ -600,7 +600,13 @@ const useHeaderMenuItems = (
           metadata: {
             include: useEda ? [VectorBase] : [],
           },
-          items: mapMenuItems ?? [],
+          items: mapMenuItems ?? [
+            {
+              key: 'maps-loading',
+              type: 'custom',
+              display: <Loading radius={4} />,
+            },
+          ],
```

Comment:
For clarity, is this loading indicator displayed only when the map is clicked from the site header?

Reply:
Sorry, no. The list of studies is fetched from the backend. In some cases, that can take several seconds, and if the user expands the "Interactive maps" menu item, it will be empty. This change renders a loading indicator.